entity-framework/core/extensions/index.md

```diff
@@ -54,6 +54,14 @@ Entity Framework Core Generator (efg) is a .NET Core CLI tool that can generate
 
 ## Extensions
 
+### Microsoft.EntityFrameworkCore.DistributedCache
+An extension for caching results of queries into the Redis cache.
+This library implements asynchronous and synchronous execution.
+It supports distributed lock to be able to use cache between n-numbers of servers which runs the application and doesn’t allow any race condition for cache update.
```

Comment:
I believe this is easier to read, but I hope I am not changing the meaning of the sentence. @grinay what do you think?
```suggestion
It supports distributed lock to be able to use cache across multiple server which run the application and avoid race conditions on cache update.
```

Reply:
That's great. Thanks for correcting my English.